src/Disks/BlobStorage/BlobStorageAuth.h

```diff
@@ -7,8 +7,26 @@
 #if USE_AZURE_BLOB_STORAGE
 
 #include <Disks/IDiskRemote.h>
+
+#if defined(__clang__)
```

Comment:
You just write

target_include_directories(... PUBLIC SYSTEM

instead of all this mess.

Reply:
CC @kssenii

Reply:
Cool. And it should be `target_include_directories(... SYSTEM PUBLIC ... `   https://github.com/ClickHouse/ClickHouse/pull/32948